Version.cmake

```diff
@@ -59,10 +59,10 @@
 # for release mode and set the version.
 #
 
-SET(Trilinos_VERSION 13.5)
-SET(Trilinos_MAJOR_VERSION 13)
-SET(Trilinos_MAJOR_MINOR_VERSION 130500)
-SET(Trilinos_VERSION_STRING "13.5 (Dev)")
+SET(Trilinos_VERSION 14.1)
```

Comment:
This looks like a bad idea, what will be in develop after 14.0 is not really what will end up in 14.1
I would suggest labeling develop as 140099, the latest (and unlikely) patch number before 14.1

Reply:
See for example the recent list of release versions: https://github.com/trilinos/Trilinos/releases